returnn/tf/util/basic.py

```diff
@@ -1339,6 +1340,142 @@ def __call__(self, shape, dtype=None, partition_info=None):
   return LoadTxtFileInitializer()
 
 
+def greenwood_function(x, scaling_constant=165.4, constant_of_integration=0.88, slope=2.1):
+  """
+  Greenwood function, convert fractional length to frequency, see
+  Schlueter, Ralf, et al. "Gammatone features and feature combination for large vocabulary speech recognition."
+  ICASSP 2007 and also
+  https://en.wikipedia.org/wiki/Greenwood_function
+  The default values are taken from RASR, see https://github.com/rwth-i6/rasr/blob/master/src/Signal/GammaTone.cc
+  :param float x:
+  :param float scaling_constant: A in [1]
+  :param float constant_of_integration: k in [1]
+  :param float slope: a in [1]
+  :return: frequency corresponding to given fractional length
+  :rtype: float
+  """
+  assert 0 <= x <= 1, "fractional length has to be between 0 and 1"
+  return scaling_constant * (np.power(10.0, slope * x) - constant_of_integration)
```

Comment:
This looks like Numpy only. Then this should not be in `tf/util` but just in `util`.